HANDBOOK.md

```diff
@@ -283,25 +282,25 @@ It is your choice to build the barriers first or the map first. For this example
    * `2000` is for 2 AM
    * etc.
   
-* Below the map constants, you will see a list of the different teams with a check-box next to them. Use the check-box to enable the teams that exists in your map.
+* Below the map constants, you will see a list of the different teams with a checkbox next to them. Use the checkbox to enable the teams that exist in your map.
   
   ![image](https://github.com/CTF-handbooks/map-maker-handbook/assets/88883098/6b893f4d-ef85-460c-a67a-a2b60eb2f49b)
 
   Next, click the `Set Flag Pos` button below your first enabled team and punch the node where that team's flag will be on. This will place the flag on top of that node.
 
-  Then, click the `Zone Bounds` button below the `Set Flag Pos` button and select the team's team-zone. A team-zone is the area in which the team's players interact in during build-time before the build-time barriers drop. During build-time, players who go out of the team-zone will be teleported back to base. Even standing on the edge of the team-zone will teleport the player back to base, **so make sure to include the barriers within your team-zone too, or players will get teleported back to base if they touch the barriers during build-time!**
+  Then, click the `Zone Bounds` button below the `Set Flag Pos` button and select the team's team-zone. A team-zone is where the team's players interact during build-time before the build-time barriers drop. During build-time, players who leave the team-zone will be teleported back to base. Even standing on the edge of the team-zone will teleport the player back to base, **so make sure to include the barriers within your team-zone too, or players will get teleported back to base if they go through the barrier area during build-time!**
```

Comment:
```suggestion
  Then, click the `Zone Bounds` button below the `Set Flag Pos` button and select the team's team-zone. A team-zone is the area where the team's players interact within during build-time before the build-time barriers drop. During build-time, players who leave the team-zone will be teleported back to base. Even standing on the edge of the team-zone will teleport the player back to base, **so make sure to include the barriers within your team-zone too, or players will get teleported back to base if they touch the barriers during build-time!**
```
Bit unclear.
For the second bit, yes, I meant "touch the barriers".

Reply:
Oh, I misunderstood the sentence as the barriers should teleport. My bad